### PR TITLE
Semeval 2014 - Task 1 Hypothesis-Premise Entailment

### PR DESCRIPTION
**Pull request by blackrosedragon2**

### CODE REVIEW

File: templates/sem_eval_2014_task_1/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      \ {{premise}}, \nPredict the Entailment relation between the premise and hypothesis\
+      \ from the labels {{\"Neutral\"}}, {{\"Entailment\"}}, {{ \"Contradiction\"\
+      }} |||\n {{[\"Neutral\", \"Entailment\",  \"Contradiction\"][entailment_judgment]}}\n"
+    name: entailment_basic_1
+    reference: ''
+    task_template: false
+  91a6b1db-be59-41bd-9eea-73bb7a4e7350: !Template
+    id: 91a6b1db-be59-41bd-9eea-73bb7a4e7350
+    jinja: 'Out of {{"neither entails nor contradicts"}}, {{"entails"}}, {{ "contradicts"}},
+      the hypothesis "{{hypothesis}}" ||| {{["neither entails nor contradicts", "entails",  "contradicts"][entailment_judgment]}}
```
Comment (craffel) [inline templates/sem_eval_2014_task_1/templates.yaml:37]: It's best to only have the model predict one thing - I'm not sure the model will be able to predict the entailment relationship if it's not given the premise.

File: templates/sem_eval_2014_task_1/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      '
+    name: entailment_localization_1
+    reference: ''
+    task_template: false
+  a58fe8b4-f185-46a9-8fca-6dc66d0812be: !Template
+    id: a58fe8b4-f185-46a9-8fca-6dc66d0812be
+    jinja: "Given the following hypothesis: {{hypothesis}}.\nAs well as the premise:\
+      \ {{premise}}, \nGive a score on how related the hypothesis and premise was,\
+      \ from the scale 1 to 5, where  1 is completely unrelated and 5 is very related:\
+      \ |||       {{relatedness_score}}\n\n"
```
Comment (craffel) [inline templates/sem_eval_2014_task_1/templates.yaml:49]: Some of these relatedness scores have a silly number of decimal places due to floating point errors. Can you do`{{((10*relatedness_score)|round)/10}}` to round to nearest first decimal place?

**Review by craffel — CHANGES_REQUESTED** [review]
Thanks, requesting a few changes. Can you add at least one task format that doesn't mention entailment? Like "Does {{premise}} imply that {{hypothesis}}? Please answer yes, no, or unclear."

File: templates/sem_eval_2014_task_1/templates.yaml (hunk 1 above)
Comment (blackrosedragon2) [reply templates/sem_eval_2014_task_1/templates.yaml:37]: You're right, I'll modify this.

File: templates/sem_eval_2014_task_1/templates.yaml (hunk 2 above)
Comment (blackrosedragon2) [reply templates/sem_eval_2014_task_1/templates.yaml:49]: Sure!

**Comment by craffel** [comment]
Thanks!

**Pull request merged**